### PR TITLE
Made QQube Domain Parameter Names More Meaningful

### PR DESCRIPTION
**Pull request by fdamken**
Depends on #74.

### CODE REVIEW

**Review by famura — CHANGES_REQUESTED** [review]
nice. I think we should also renome the other domain parameters for consistency

File: Pyrado/scripts/training/qq-su_npdr.py [hunk 1]
```diff
@@ -99,25 +99,36 @@
...
-    # dp_mapping = {0: "Dr", 1: "Dp", 2: "Rm", 3: "km"}
-    # dp_mapping = {0: "Rm", 1: "km", 2: "Mr", 3: "Mp"}
-    # dp_mapping = {0: "Dr", 1: "Dp", 2: "Rm", 3: "km", 4: "Mr", 5: "Mp", 6: "Lr", 7: "Lp", 8: "g"}
-    dp_mapping = {0: "Dr", 1: "Dp", 2: "Rm", 3: "km", 4: "Mr", 5: "Mp", 6: "Lr", 7: "Lp", 8: "g", 9: "act_delay"}
+    # dp_mapping = {0: "voltage_thold_neg", 1: "voltage_thold_pos"}
+    # dp_mapping = {0: "voltage_thold_neg", 1: "voltage_thold_pos", 2: "act_delay"}
+    # dp_mapping = {0: "damping_rot_pole", 1: "damping_pend_pole"}
+    # dp_mapping = {0: "damping_rot_pole", 1: "damping_pend_pole", 2: "motor_resistance", 3: "motor_back_emf"}
+    # dp_mapping = {0: "motor_resistance", 1: "motor_back_emf", 2: "mass_rot_pole", 3: "mass_pend_pole"}
+    # dp_mapping = {0: "damping_rot_pole", 1: "damping_pend_pole", 2: "motor_resistance", 3: "motor_back_emf", 4: "mass_rot_pole", 5: "mass_pend_pole", 6: "length_rot_pole", 7: "length_pend_pole", 8: "gravity_const"}
```
Comment (famura) [inline Pyrado/scripts/training/qq-su_npdr.py:107]: nice that you also got the commented stuff

**Review by famura — COMMENTED** [review]
Please wait with merging

File: Pyrado/scripts/training/qq-su_npdr.py (hunk 1 above)
Comment (fdamken) [reply Pyrado/scripts/training/qq-su_npdr.py:107]: :)

**Comment by famura** [comment]
@fdamken I am nor ready to merge this PR after it has been rebased

File: Pyrado/pyrado/environments/quanser/servers/ball_balancer/calc_conversion_constants.m [hunk 2]
```diff
@@ -27,8 +27,8 @@
     K_LBS2N = 4.4482216;
     % Newton to Pound (lbs/N/)
     K_N2LBS = 1 / K_LBS2N;
-    % from g to m/s^2
+    % from gravity_const to m/s^2
     K_G2MS = 9.81;
-    % from m/s^2 to g
+    % from m/s^2 to gravity_const
```
Comment (famura) [inline Pyrado/pyrado/environments/quanser/servers/ball_balancer/calc_conversion_constants.m:32]: please undo this

File: Pyrado/pyrado/environments/pysim/pendulum.py [hunk 3]
```diff
@@ -93,22 +93,24 @@ def observe(self, state) -> np.ndarray:
...
     def get_nominal_domain_param(cls) -> dict:
         return dict(
-            g=9.81,  # gravity constant [m/s**2]
+            gravity_const=9.81,  # gravity constant [m/s**2]
             m_pole=1.0,  # mass of the pole [kg]
             l_pole=1.0,  # half pole length [m]
             d_pole=0.05,  # rotational damping of the pole [kg*m**2/s]
             tau_max=3.5,
-        )  # maximum applicable torque [N*m] (under-actuated if < m*l*g/2)
+        )  # maximum applicable torque [N*m] (under-actuated if < m*l*gravity_const/2)
```
Comment (famura) [inline Pyrado/pyrado/environments/pysim/pendulum.py:101]: please undo this

File: Pyrado/pyrado/environments/quanser/servers/ball_balancer/config_2dbb.m [hunk 4]
```diff
@@ -9,7 +9,7 @@
 % r_arm     Distance between SRV02 output gear shaft and 
 %           coupled joint                                       (m)
 % J_b       Moment of inertia of ball.                          (kg.m^2)
-% g         Gravitational constant                              (m/s^2)
+% gravity_const         Gravitational constant                              (m/s^2)
```
Comment (famura) [inline Pyrado/pyrado/environments/quanser/servers/ball_balancer/config_2dbb.m:12]: please undo this

File: Pyrado/pyrado/environments/quanser/servers/ball_balancer/config_2dbb.m [hunk 5]
```diff
@@ -19,15 +19,15 @@
 % Quanser Consulting Inc.
 %%
 % 
-function [ L_tbl, r_arm, r_b, m_b, J_b, g, THETA_MIN, THETA_MAX] = config_2dbb( )
+function [ L_tbl, r_arm, r_b, m_b, J_b, gravity_const, THETA_MIN, THETA_MAX] = config_2dbb( )
```
Comment (famura) [inline Pyrado/pyrado/environments/quanser/servers/ball_balancer/config_2dbb.m:22]: please undo this

File: Pyrado/pyrado/environments/quanser/servers/ball_balancer/config_2dbb.m [hunk 6]
```diff
@@ -19,15 +19,15 @@
...
     % Calculate useful conversion factors
     [ K_R2D, K_D2R, K_IN2M, K_M2IN, K_RDPS2RPM, K_RPM2RDPS, K_OZ2N, K_N2OZ, K_LBS2N, K_N2LBS, K_G2MS, K_MS2G ] = calc_conversion_constants ();    
     % Table width and length (m)
     L_tbl = 27.5 / 100; 
     % Distance between SRV02 output gear shaft and coupled joint (m)
     r_arm = 1 * K_IN2M;
     % Gravitational constant (m/s^2)
-    g = 9.81;
+    gravity_const = 9.81;
```
Comment (famura) [inline Pyrado/pyrado/environments/quanser/servers/ball_balancer/config_2dbb.m:30]: please undo this

File: Pyrado/pyrado/environments/quanser/servers/ball_balancer/config_srv02.m [hunk 7]
```diff
@@ -13,9 +13,9 @@
 %
 % ************************************************************************
 % Output parameters:
-% Rm        Motor armaturce resistance                          (ohm)
+% motor_resistance        Motor armaturce resistance                          (ohm)
 % kt        Motor torque constant                               (N.m/A)
-% km        Motor back-EMF constant                             (V.s/rad)
+% motor_back_emf        Motor back-EMF constant                             (V.s/rad)
```
Comment (famura) [inline Pyrado/pyrado/environments/quanser/servers/ball_balancer/config_srv02.m:18]: please undo this

File: Pyrado/pyrado/environments/quanser/servers/ball_balancer/d_2dbb_model_param.m [hunk 8]
```diff
@@ -11,7 +11,7 @@
 % r_b       Radius of ball                                      (m)
 % m_b       Mass of ball                                        (kg)
 % J_b       Moment of inertia of ball.                          (kg.m^2)
-% g         Gravitational constant                              (m/s^2)
+% gravity_const         Gravitational constant                              (m/s^2)
```
Comment (famura) [inline Pyrado/pyrado/environments/quanser/servers/ball_balancer/d_2dbb_model_param.m:14]: please undo this

File: Pyrado/pyrado/environments/quanser/servers/ball_balancer/d_2dbb_model_param.m [hunk 9]
```diff
@@ -21,8 +21,8 @@
 % Quanser Consulting Inc.
 %%
 %
-function [ K_bb ] = d_2dbb_model_param(r_arm, L_tbl, r_b, m_b, J_b, g)
+function [ K_bb ] = d_2dbb_model_param(r_arm, L_tbl, r_b, m_b, J_b, gravity_const)
     % Model gain (m/s^2/rad)
-%     K_bb = 2 * m_b * g * r_arm * r_b^2 / L_tbl / ( m_b * r_b^2 + J_b );
-    K_bb = 6/5*g*r_arm/L_tbl;
+%     K_bb = 2 * m_b * gravity_const * r_arm * r_b^2 / L_tbl / ( m_b * r_b^2 + J_b );
+    K_bb = 6/5*gravity_const*r_arm/L_tbl;
```
Comment (famura) [inline Pyrado/pyrado/environments/quanser/servers/ball_balancer/d_2dbb_model_param.m:27]: please undo this

File: Pyrado/pyrado/environments/quanser/servers/ball_balancer/d_model_param.m [hunk 10]
```diff
@@ -5,9 +5,9 @@
 %
 % ************************************************************************
 % Input parameters:
-% Rm        Motor armaturce resistance                          (ohm)
+% motor_resistance        Motor armaturce resistance                          (ohm)
 % kt        Motor torque constant                               (N.m/A)
-% km        Motor back-EMF constant                             (V.s/rad)
+% motor_back_emf        Motor back-EMF constant                             (V.s/rad)
```
Comment (famura) [inline Pyrado/pyrado/environments/quanser/servers/ball_balancer/d_model_param.m:10]: please undo this

File: Pyrado/pyrado/environments/quanser/servers/ball_balancer/d_model_param.m [hunk 11]
```diff
@@ -23,7 +23,7 @@
 % Quanser Consulting Inc.
 %%
 %
-function [K,tau] = d_model_param(Rm, kt, km, Kg, eta_g, Beq, Jeq, eta_m, daqb)        
+function [K,tau] = d_model_param(motor_resistance, kt, motor_back_emf, Kg, eta_g, Beq, Jeq, eta_m, daqb)
```
Comment (famura) [inline Pyrado/pyrado/environments/quanser/servers/ball_balancer/d_model_param.m:26]: please undo this

File: Pyrado/pyrado/environments/quanser/servers/ball_balancer/setup_2dbb.m [hunk 12]
```diff
@@ -40,11 +40,11 @@
...
 %% System Parameters
 % Sets model variables according to the user-defined system configuration
-[ Rm, kt, km, Kg, eta_g, Beq, Jm, Jeq, eta_m, K_POT, K_TACH, K_ENC, VMAX_AMP, IMAX_AMP ] = config_srv02( EXT_GEAR_CONFIG, ENCODER_TYPE, TACH_OPTION, AMP_TYPE, LOAD_TYPE );
+[ motor_resistance, kt, motor_back_emf, Kg, eta_g, Beq, Jm, Jeq, eta_m, K_POT, K_TACH, K_ENC, VMAX_AMP, IMAX_AMP ] = config_srv02( EXT_GEAR_CONFIG, ENCODER_TYPE, TACH_OPTION, AMP_TYPE, LOAD_TYPE );
 % Load 2DBB model parameters.
-[ L_tbl, r_arm, r_b, m_b, J_b, g, THETA_MIN, THETA_MAX ] = config_2dbb( );
+[ L_tbl, r_arm, r_b, m_b, J_b, gravity_const, THETA_MIN, THETA_MAX ] = config_2dbb( );
 % Load model parameters based on SRV02 configuration.
-[ K, tau ] = d_model_param(Rm, kt, km, Kg, eta_g, Beq, Jeq, eta_m, AMP_TYPE);
+[ K, tau ] = d_model_param(motor_resistance, kt, motor_back_emf, Kg, eta_g, Beq, Jeq, eta_m, AMP_TYPE);
```
Comment (famura) [inline Pyrado/pyrado/environments/quanser/servers/ball_balancer/setup_2dbb.m:47]: please undo this

File: Pyrado/pyrado/environments/quanser/servers/ball_balancer/setup_2dbb.m [hunk 13]
```diff
@@ -61,7 +61,7 @@
     %
 elseif strcmp ( CONTROL_TYPE , 'AUTO' )
     % Calculate Balance Table model gain.
-    [ K_bb ] = d_2dbb_model_param(r_arm, L_tbl, r_b, m_b, J_b, g);
+    [ K_bb ] = d_2dbb_model_param(r_arm, L_tbl, r_b, m_b, J_b, gravity_const);
```
Comment (famura) [inline Pyrado/pyrado/environments/quanser/servers/ball_balancer/setup_2dbb.m:64]: please undo this

File: Pyrado/pyrado/environments/quanser/servers/cartpole/config_ip02.m [hunk 14]
```diff
@@ -6,7 +6,7 @@
 % and finally returns the calculated model parameters of the IP02 Quanser plant.
 %
 % IP02 system nomenclature:
-% Rm        Motor Armature Resistance                                   (Ohm)
+% motor_resistance        Motor Armature Resistance                                   (Ohm)
```
Comment (famura) [inline Pyrado/pyrado/environments/quanser/servers/cartpole/config_ip02.m:9]: please undo this

File: Pyrado/pyrado/environments/quanser/servers/cartpole/config_ip02.m [hunk 15]
```diff
@@ -27,21 +27,21 @@
...
 
 %% Calculate the IP01 or IP02 model parameters 
-function [ Rm, Jm, Kt, eta_m, Km, Kg, eta_g, M, r_mp, Beq ] = calc_IP02_parameters( IP02_LOAD_TYPE, AMP_TYPE )
+function [ motor_resistance, Jm, Kt, eta_m, Km, Kg, eta_g, M, r_mp, Beq ] = calc_IP02_parameters( IP02_LOAD_TYPE, AMP_TYPE )
 global K_IN2M K_D2R K_RDPS2RPM K_OZ2N
 % Set these variables (used in Simulink Diagrams)
 global VMAX_AMP IMAX_AMP ALPHA_MAX ALPHA_MIN
 % Motor Armature Resistance (Ohm)
-Rm = 2.6;
+motor_resistance = 2.6;
```
Comment (famura) [inline Pyrado/pyrado/environments/quanser/servers/cartpole/config_ip02.m:44]: please undo this

File: Pyrado/pyrado/environments/quanser/servers/cartpole/setup_ip02_sip.m [hunk 16]
```diff
@@ -69,23 +69,23 @@
...
-% [Er, a_max] = d_swing_up(eta_m, eta_g, Kg, Kt, Rm, r_mp, Jeq, Mp, lp);
+% [Er, a_max] = d_swing_up(eta_m, eta_g, Kg, Kt, motor_resistance, r_mp, Jeq, mass_pend_pole, lp);
 % epsilon = 10.0 * pi / 180;
 
 % For the following state vector: X = [ xc; alpha; xc_dot; alpha_dot ]
 % Initialization of the State-Space Representation of the Open-Loop System
 % Call the following Maple-generated file to initialize the State-Space Matrices: A, B, C, and D
 % ABCD Eqns relative to Fc
-% [ A, B, C, D ] = SIP_ABCD_eqns(Rm, Kt, eta_m, Km, Kg, eta_g, Jeq, Mp, Bp, lp, g, Jp, r_mp, Beq);
+% [ A, B, C, D ] = SIP_ABCD_eqns(motor_resistance, Kt, eta_m, Km, Kg, eta_g, Jeq, mass_pend_pole, Bp, lp, gravity_const, Jp, r_mp, Beq);
```
Comment (famura) [inline Pyrado/pyrado/environments/quanser/servers/cartpole/setup_ip02_sip.m:88]: please undo this

**Review by famura — CHANGES_REQUESTED** [review]
Sorry, I forgot to tell you that the matlab files should not be changed

**Pull request merged**